### ISSUE TITLE
Crash while trying to render the logistics / directions (futurepack.client.render.block.RenderLogistic.renderLogistics)

### ISSUE THREAD

**Issue by lephro** [mcenderdragon/Futurepack-API#827]
**To Reproduce**
Took out the logistics tool for the very first time and switched through the modes as it crashed on me. 
I cannot connect to the server anymore aswell... it keeps crashing :/

Seems that it has some issues with the directions in the logistic mode. 

Mod was a lot of fun until then. Was minutes before i wanted to do my first "space flight". 
I'll see if i can restore a backup or something, to make myself able join again :/

Give me a sign if you need any additional infos.

**Version**
Minecraft Version: 1.16.5
Forge Version: 36.2.2
Futurepack Version: 31.2.6976

**Logs**
Could not upload to pastebin for some reason:

[hs_err_pid4584.log](https://github.com/mcenderdragon/Futurepack-API/files/7391421/hs_err_pid4584.log)
[hs_err_pid12316.log](https://github.com/mcenderdragon/Futurepack-API/files/7391422/hs_err_pid12316.log)

**Comment by mcenderdragon** [comment]
update your java version you are using the old default launche rone (java 8, 1.8.51)

**Comment by lephro** [comment]
Thats the default vanilla mc java version, that the launcher trys to enforce on you.
I also got a Java14 profile set up. I will try it with that one too, but it generally tends to crash once in a while, so that I go back to vanilla Java. 

Is there something I'm missing? 
Or anything particular that you would suggest to use? 

Thank you a lot in advance and generally thank you for making this mod btw. I feel like its very underappreciated. 

As a side note: I feel like that the Name Futurepack is very generic and gives ppl the wrong suggestion, when they read the nam ein the list. Took me some time to find it myself... glad I did tho... you got 16 stars now ;)
